### PR TITLE
AMG: commands update for new API version 

### PR DESCRIPTION
**Pull request by yugangw-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
az grafana api-key
az grafana update

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
@kairu-ms for awareness

File: src/amg/azext_amg/tests/latest/test_amg_livescenario.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class AmgLiveScenarioTest(LiveScenarioTest):
 
     @AllowLargeResponse(size_kb=3072)
     @ResourceGroupPreparer(name_prefix='cli_test_amg', location='westeurope')
-    def test_amg_e2e(self, resource_group):
+    def xxx_test_amg_e2e(self, resource_group):
```
Comment (kairu-ms) [inline src/amg/azext_amg/tests/latest/test_amg_livescenario.py:21]: It looks like a strange test name.

File: src/amg/setup.py [hunk 2]
```diff
@@ -16,10 +16,8 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-
 VERSION = '0.1.3'
```
Comment (kairu-ms) [inline src/amg/setup.py:19]: Do you need to update version for release?

File: src/amg/azext_amg/tests/latest/test_amg_livescenario.py (hunk 1 above)
Comment (yugangw-msft) [reply src/amg/azext_amg/tests/latest/test_amg_livescenario.py:21]: Good catch. This file is meant to be deleted as I have consolidated the e2e test in a single file. Let me remove it right now

File: src/amg/setup.py (hunk 2 above)
Comment (yugangw-msft) [reply src/amg/setup.py:19]: Let us do it later so to align with other release activity

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**